plugins/sqlfluff-templater-dbt/test/templater_test.py

```diff
@@ -70,10 +72,24 @@ def test__templater_dbt_templating_result(
         fname=os.path.join(project_dir, "models/my_new_project/", fname),
         config=FluffConfig(configs=DBT_FLUFF_CONFIG),
     )
-    assert (
-        str(templated_file)
-        == open("plugins/sqlfluff-templater-dbt/test/fixtures/dbt/" + fname).read()
+    template_output_folder_path = (
+        "plugins/sqlfluff-templater-dbt/test/fixtures/dbt/templated_output/"
     )
+    dbt_version_specific_fixture_folder = {(1, 0): "dbt_utils_0.8.0"}.get(
+        DBT_VERSION_TUPLE
+    )
+    fixture_path = None
+    if dbt_version_specific_fixture_folder:
+        version_specific_path = (
+            Path(template_output_folder_path)
+            / dbt_version_specific_fixture_folder
+            / fname
+        )
+        if os.path.exists(version_specific_path):
+            fixture_path = version_specific_path
+    if not fixture_path:
+        fixture_path = template_output_folder_path + fname
+    assert str(templated_file) == open(fixture_path).read()
```

Comment:
Following up the above comment about `Path` vs `os.path`, if we were using `Path` here, we could use [the `read_text()` function](https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text), which opens, reads, and closes the file in one step.